packages/app/src/elements/item-view.ts

```diff
@@ -226,6 +236,28 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         }
     }
 
+    private _addExpirationDefault() {
+        if (!this._item) {
+            return;
+        }
+
+        this._item.expiresBy = add(new Date(), { days: 7 });
+        this._item.expiryFrequency = ExpiryFrequencyOption.Yearly;
+
+        this.save();
+    }
+
+    private _removeExpiration() {
+        if (!this._item) {
+            return;
+        }
+
+        this._item.expiresBy = undefined;
+        this._item.expiryFrequency = undefined;
+
+        this.save();
+    }
```

Comment:
@MaKleSoft I made this to test out the report quickly, but I think there might be something to it, on making it as easy as clicking a button to have an item expire yearly, which is the most common frequency. Let me know your thoughts.